### ISSUE TITLE
Settings editor doesn't respect ctrl+up and ctrl+down

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode#188754]
Testing #188536

Not sure if the settings editor is intended to support this, but it seems very similar to the other places that do support it.

**Comment by weartist** [comment]
@alex I'll try to fix that, if he should support it

**Comment by weartist** [comment]
@alexr00 I'll try to fix that, if it should be supported
